_posts/2024/2024-02-06-about-caching-in-eo.md

```diff
@@ -0,0 +1,237 @@
+---
+layout: post
+date: 2024-02-06
+title: "Build cache in EO and other build systems"
+author: Alekseeva Yana
+---
+
+
+## Introduction 
+In [EO](https://github.com/objectionary/eo) a caching is used to speed up program execution.
+While developing [EO](https://github.com/objectionary/eo) we found a caching 
+[error](https://github.com/objectionary/eo/issues/2790) in `eo-maven-plugin`
+for EO version `0.34.0`. The error occurred because the cache was searched for the needed file using 
+a comparison of compilation time and caching time.
+This is not the most reliable verification method,
+because caching time does not have to be equal to compilation time.
+[Unit tests](https://github.com/objectionary/eo/pull/2749) were written to show that the
+cache does not work correctly. Additionally, reading a file was necessary to obtain a program name 
+that slowed down the build process. 
+That we came to the conclusion that we need caching with a reliable verification method
+that does not require reading a file system. Using a cache should save us enough time for building a project.
```

Comment:
@Yanich96 

> Using a cache should save us enough time for building a project.

We already use some cache. So it already saves some time.
btw, "enough" - How much is that?